Packer/ArkCase-FOIA/startup.sh

```diff
@@ -43,6 +44,10 @@ if [ ! -e /var/lib/admin-password-changed ]; then
     # Update the password in the portal configuration
     sed -i "19s/@rKc@3e/$clear_password/g" "${rootdir}/data/arkcase-home/.arkcase/acm/acm-config-server-repo/arkcase-portal-server.yaml"
 
+    # Update the password in the foia analytical reports configuration
+    sed -i "s/ARKCASE_PASS=.*/ARKCASE_PASS=$clear_password/g" "${rootdir}/install/pentaho/foia-reports-dw-2021.01/config/arkcase_config.properties"
```

Comment:
What is `2021.01` ? Should it be the version variable?

Reply:
Yes, I have defined it in the script but somehow I forgot to use it here. Now it is ok :)